spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -755,6 +763,45 @@ def setup_and_check(self, data, title='', readonly=False,
         self.stack.currentChanged.connect(self.current_widget_changed)
         self.layout.addWidget(self.stack, 1, 0)
 
+        # ---- Toolbar and actions
+        toolbar = SpyderToolbar(parent=None, title='Editor toolbar')
```

Comment:
Why `parent=None` and not `parent=self`?

Reply:
Fixed (the cause was that I copied from the Collection Editor).